### ISSUE TITLE
Still working on this?

### ISSUE THREAD

**Issue by julianvdh** [NukeAlchemist/blupi#3]
Hello there,

Are you still working on this?
It would be really nice to see this finished and ready to use

**Comment by NukeAlchemist** [comment]
Nah. Unfortunately the hardware necessary to make this work doesn’t really exist—the antennas that *could* work are both prohibitively expensive and consume a great deal of power. 

---
Life, E. K.

**Comment by raymiec** [comment]
@NukeAlchemist this actually kinda works ok in the UK, we only use Tetra here between 380 and 385 as does most probably the rest of the EU (albeit on different frequencies). Fire Brigade and Ambulance service seem to emit a constant tetra signal, police seem to be more intermittent.

I've updated your python script a bit to work with python 3, added sound / light notifications. 

Initial tests on a raspberry pi 3b, RTL-SDR dongle. ran fine.

For testing I did have to change the frequency as I live right next to a Police Station, so I'm flooded with Tetra signals.

I'll get out over the weekend and give it a proper test and report back.

**Comment by Lauwy222** [comment]
@raymiec update?

**Comment by raymiec** [comment]
Not actually worked on it in a while, i did make a nice little system with a pi zero 2 and mmdvm hotspot.  Also wrote an android app that would link up to it via Bluetooth, android app you are able to hook up a sdr directly as well.

The way I done it, it worked out you could only really pick up the cops whilst they were actually transmitting. Still useful... but with cheap hardware I was never able to detect the tetra bursts that the radios send periodically.  I might get back into it again soon.

**Comment by Lauwy222** [comment]
Allright, please ping me if u start working on it again. My team is also working on a scanner atm.

**Comment by albielew** [comment]
@raymiec If it helps in the UK the handset to base is 380Mhz - 385Mhz and each handset sends a "sync" signal every 4 seconds
You don't need an expensive antenna or anything like  @NukeAlchemist  seems to think, a simple piece of wire of around 195mm as an antenna
You'll only be able to detect a short range but that's ideal as you don't want it pinging you continually, only when one of the emergency services are getting near to you (or going away)

Do you have any details on how you got it working?
I'd like to have a Pi with a beeper and maybe either a display that shows my speed from a gps module as well as display of received signal strength and the beeper go off if it got above a set signal strength

the main problem is the signals can be anywhere within the 380-385Mhz range, so you either need an sdr with a wide bandwidth, or 2 rtl dongles connected at the same time, each scanning 2.5Mhz bandwitch continually.
